microsoft-edge/webview2/samples/wv2deploymentwixcustomactionsample.md

```diff
@@ -166,7 +166,7 @@ If you want to package the Evergreen WebView2 Runtime Standalone Installer with
 
 1. If you're targeting non-X64 devices, you may also want to edit the `MicrosoftEdgeWebView2RuntimeInstallerX64` filename to reflect the correct architecture.
 
-1. Under `<!-- Step 5: Config execute sequence of custom action -->`, uncomment the `<Custom Action='InvokeStandalone' ...>` element below `<!-- [Package Standalone Installer] ...-->`.
+1. Under `<!-- Step 5: Config run sequence of custom action -->`, uncomment the `<Custom Action='InvokeStandalone' ...>` element below `<!-- [Package Standalone Installer] ...-->`.
```

Comment:
```suggestion
1. Under `<!-- Step 5: Config execute sequence of custom action -->`, uncomment the `<Custom Action='InvokeStandalone' ...>` element below `<!-- [Package Standalone Installer] ...-->`.
```
Reverting b/c the xml comment is from the code listing
https://github.com/MicrosoftEdge/WebView2Samples/blob/main/SampleApps/WV2DeploymentWiXCustomActionSample/Product.wxs - line 64 & 65; 65 is `<InstallExecuteSequence>`

An xml comment in the code repo is out of scope; would need to create a PR in that code repo before (or simult. w/) changing here in docs repo. Such a PR might get rejected, since the xml code itself uses the word/phrase 'ExecuteSequence': line 65 is `<InstallExecuteSequence>`.

Style guide doesn't here discuss Dev coding docs:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/execute
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/run-vs-execute